test/WebSites/BasicWebSite/Areas/Area1/Views/RemoteAttribute_Home/Details.cshtml

```diff
@@ -64,13 +64,5 @@
 </div>
 
 <p>
-    @Html.ActionLink("Create a new one", "Create") |
-    @if (string.Equals("Aria", areaName, StringComparison.OrdinalIgnoreCase))
-    {
-        @Html.RouteLink("Go back to home", "Home")
```

Comment:
FYI I think this was more about navigating around site in a browser than a test addition. As a sample, we want the site to be human-usable even more.